draft-ietf-rats-architecture.md

```diff
@@ -123,6 +123,10 @@ Attester:
 
 : An entity whose attributes must be appraised in order to determine whether the entity is considered trustworthy, such as when deciding whether the entity is authorized to perform some operation
 
+Claim:
+: A piece of information asserted about a subject. (Compare /claim/ in {{RFC7519}})
```

Comment:
"A piece of asserted information, often in the form of a name/value pair."

Reply:
Thomas Hardjono
This is what th SAML standard says: An assertion is a package of information that supplies zero or more statements made (about subject)

Reply:
I don't feel strongly about using 'subject' but it isn't used anywhere else whereas we use 'entity' elsewhere. If the reader knows we meant 'entity' even though we used 'subject' then it is OK. However, it might be just as easy to say entity and not introduce the question in the readers mind whether subject has special meaning or not. We use entity 29 times already. We use 'subject' once but for a different meaning.